graphql2/gqlgen.yml

```diff
@@ -30,6 +30,8 @@ models:
     model: github.com/target/goalert/alert.Alert
   AlertState:
     model: github.com/target/goalert/alert.State
+  AlertLog:
+    model: github.com/target/goalert/alert.Log  
```

Comment:
Should this be `log.Entry` instead of `alert.Log`? I think the `alert.Log` is only intended for "already acknowledged" type things.